src/formatter/date.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  DateFromString,
+  DateFromTimestamp,
+  UtcOnlyIsoDateFromString
+} from "@pagopa/ts-commons/lib/dates";
+import * as E from "fp-ts/Either";
+import { identity, pipe } from "fp-ts/function";
+import * as t from "io-ts";
+import { failure } from "io-ts/PathReporter";
+
+const onLeft = (errors: t.Errors): string =>
+  pipe(failure(errors), e => e.join("\n"));
```

Comment:
WWe can safely remove this utility method since we can handle `Error` by using `Either`